packages/strapi-admin/ee/admin/containers/Roles/CreatePage/PluginsAndSettings/Row/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo } from 'react';
+import { Flex, Text } from '@buffetjs/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { PermissionsWrapper, RowContainer } from 'strapi-helper-plugin';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+// import SubCategory from 'ee_else_ce/components/Roles/Permissions/PluginsAndSettingsPermissions/PermissionRow/SubCategory';
+import SubCategory from '../SubCategory';
+import RowStyle from './Wrapper';
+
+const PermissionRow = ({
+  childrenForm,
+  kind,
+  name,
+  isOpen,
+  isWhite,
+  onOpenCategory,
+  pathToData,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const handleClick = () => {
+    onOpenCategory(name);
+  };
+
+  const categoryName = useMemo(() => {
+    const split = name.split('::');
+
+    return split[split.length - 1];
```

Comment:
Nitpicky: when you don't really care about mutating the array, you can simply rely on `split.pop()` 😊

Reply:
fixed in #9420